libjsonutils/cmake/FindRapidJSON.cmake

```diff
@@ -1,46 +1,55 @@
 #FindRapidJSON.cmake
-# 
+#
 # Finds the rapidjson library
 #
+# from http://rapidjson.org/
+#
 # This will define the following variables
 #
-#    RapidJSON_FOUND
-#    RapidJSON_INCLUDE_DIRS
+#    RAPIDJSON_FOUND
+#    RAPIDJSON_VERSION
+#    RAPIDJSON_INCLUDE_DIRS
 #
 # and the following imported targets
 #
 #     RapidJSON::RapidJSON
 #
 # Author: Pablo Arias - pabloariasal@gmail.com
+#
+# clausklein$ cmakelint *.cmake
+# FindRapidJSON.cmake:0: Find modules should use uppercase names;
+#   consider using FindRAPIDJSON.cmake [convention/filename]
+# Total Errors: 1
 
-find_package(PkgConfig)
+find_package(PkgConfig REQUIRED)
```

Comment:
I don't think this makes sense. You're failing the entire configure even for a non-required `RapidJSON` finding.

Reply:
removed